lib/url.js

```diff
@@ -377,7 +377,7 @@ Url.prototype.parse = function parse(url, parseQueryString, slashesDenoteHost) {
     }
 
     // pull out port.
-    this.parseHost();
+    this.parseHost(url);
```

Comment:
Is this correct? `this.parseHost()` does not take a parameter

Reply:
Whoops. That was leftover from a previous attempt at solving this. Removed.